test/util/bitcoin-util-test.py

```diff
@@ -20,40 +20,44 @@
 import subprocess
 import sys
 
+
 def main():
     config = configparser.ConfigParser()
-    config.read_file(open(os.path.dirname(__file__) + "/../config.ini"))
+    config.read_file(open(os.path.join(os.path.dirname(__file__), "../config.ini")))
 
     parser = argparse.ArgumentParser(description=__doc__)
-    parser.add_argument('-v', '--verbose', action='store_true')
+    parser.add_argument("-v", "--verbose", action="store_true")
```

Comment:
I don't it is worth the overhead to change apostrophe to quotation mark.

Reply:
agree (and for short symbol-like strings as in this line, single quotes seem to be most people's preferred style)